docs/user/openstack/README.md

```diff
@@ -122,30 +157,48 @@ platform:
     cloud:            standalone
     externalNetwork:  public
     region:           regionOne
-    computeFlavor:    m1.medium
-    lbFloatingIP:     "10.19.115.117"
+    computeFlavor:    m1.large
+    lbFloatingIP:     "<API FIP>"
 ```
 
-This will let you do a fully unattended end to end deployment.
+At the time of writing, you will have to create a second floating ip and attach it to the ingress-port if you want to be able to reach *.apps externally.
+This can be done after the install completes in three steps:
+
+Get the ID of the ingress port:
 
+```sh
+openstack port list | grep "ingress-port"
+```
 
-### No Floating IP
+Create and associate a floating IP to the ingress port:
 
-If you don't want to pre-create a floating IP address, you will still want to create the API DNS record or the installer will fail waiting for the API.
+```sh
+openstack floating ip create --port <ingress port id> <external network>
+```
+
+Add A record in your dns for *apps. in your DNS:
+
+```
+*.apps.example.shiftstack.com  IN  A  <ingress FIP>
+```
+OR add A record in `/etc/hosts`:
 
+```
+    <ingress FIP> console-openshift-console.apps.example.shiftstack.com
```

Comment:
Perhaps we should add all the different addresses the cluster uses by default, if I'm not mistaken:
```
<ingress FIP> console-openshift-console.apps.example.shiftstack.com
<ingress FIP> integrated-oauth-server-openshift-authentication.apps.example.shiftstack.com
<ingress FIP> oauth-openshift.apps.example.shiftstack.com
<ingress FIP> prometheus-k8s-openshift-monitoring.apps.example.shiftstack.com
<ingress FIP> grafana-openshift-monitoring.apps.example.shiftstack.com
```